lib/defaults/index.js

```diff
@@ -152,7 +152,16 @@ const defaults = {
     common: {
       'Accept': 'application/json, text/plain, */*'
     }
-  }
+  },
+
+  fetcher: null,
+
+  fetchOptions: {
+    cache: 'default',
+    redirect: 'follow'
+  },
+
+  parsedUrl: null,
```

Comment:
default values for new config properties related to fetch (`fetcher`, `fetchOptions`, and `parsedUrl`)